Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9.0-buster
+
+RUN sed -i 's/deb.debian.org/mirrors.aliyun.com/g' /etc/apt/sources.list
```

Comment:
Is this setting only for Linux? How about Win and macOS?

Reply:
NO. `RUN` means **to run in the docker** but not the **host**. So there is no compatible problem. But here I just made a mistake, I will correct it later.

Reply:
Now it works correctly.